{"ff829724996e5c1d11f7d36b297be72f:2fb86ed1954d89b334b128c6ec0cf078":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"8f2f39587b6c0ce1","status":"broken","statusDetails":"Test timeout of 30000ms exceeded.","time":{"start":1741929767891,"stop":1741929798012,"duration":30121}}]},"907e9fa1146ee8ce5a0923ba0685f100:2fb86ed1954d89b334b128c6ec0cf078":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"ed39858dd8494d0e","status":"failed","statusDetails":"Error: Timed out 5000ms waiting for expect(locator).toHaveURL(expected)\n\nLocator: locator(':root')\nExpected string: \"https://traxionpay-app.web.app/#/onboarding\"\nReceived string: \"https://traxionpay-app.web.app/#/login\"\nCall log:\n  - expect.toHaveURL with timeout 5000ms\n  - waiting for locator(':root')\n    9 × locator resolved to <html>…</html>\n      - unexpected value \"https://traxionpay-app.web.app/#/login\"\n","time":{"start":1741929698548,"stop":1741929703917,"duration":5369}}]}}